docs/sources/operations/observability.md

```diff
@@ -23,7 +23,7 @@ The Loki Distributors expose the following metrics:
 | ------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------ |
 | `loki_distributor_ingester_appends_total`         | Counter     | The total number of batch appends sent to ingesters.                                                                                 |
 | `loki_distributor_ingester_append_failures_total` | Counter     | The total number of failed batch appends sent to ingesters.                                                                          |
-| `loki_distributor_bytes_received_total`           | Counter     | The total number of uncompressed bytes received per tenant.                                                                          |
+| `loki_distributor_bytes_received_total`           | Counter     | The total number of uncompressed bytes received per tenant and retention hours.                                                                          |
```

Comment:
I'm not sure my suggested change here helps clarify anything.  Use it if you think it is better.

```suggestion
| `loki_distributor_bytes_received_total`           | Counter     | The total number of uncompressed bytes received per both tenant and retention hours.                                                                          |
```